graders/c/cgrader/cgrader.py

```diff
@@ -450,7 +457,11 @@ def compile_re(t):
             comment = (
                 ""
                 if len(exp_output) == 1
-                else " all of" if must_match_all_outputs else " one of"
+                else (
+                    " all of"
+                    if must_match_all_outputs in (True, "all", "partial")
```

Comment:
Instead of doing the tuple check, on the star of the function you can do something like the following
```python
if must_match_all_outputs is True:
    must_match_all_outputs = "all"
```

Reply:
Done in fa6c7d8.